lib/java/com/google/android/material/slider/BaseSlider.java

```diff
@@ -827,11 +828,17 @@ private void createLabelPool() {
     }
 
     // Add a stroke if there is more than one label for when they overlap.
-    int strokeWidth = labels.size() == 1 ? 0 : 1;
```

Comment:
This change will break the existing behavior. Please preserve the default behavior when there's only one label, no border will be drawn.